app/models/validations/date_validations.rb

```diff
@@ -68,20 +68,32 @@ def validate_startdate(record)
 
 private
 
-  def first_collection_start_date
-    @first_collection_start_date ||= FormHandler.instance.lettings_forms["previous_lettings"].start_date
+  def previous_collection_start_suffix
+    @previous_collection_start_suffix ||= previous_collection_start_date.strftime("%y").to_i
   end
 
-  def first_collection_end_date
-    @first_collection_end_date ||= FormHandler.instance.lettings_forms["previous_lettings"].end_date
+  def current_collection_start_suffix
+    @current_collection_start_suffix ||= current_collection_start_date.strftime("%y").to_i
   end
 
-  def second_collection_start_date
-    @second_collection_start_date ||= FormHandler.instance.lettings_forms["current_lettings"].start_date
+  def previous_collection_start_date
+    @previous_collection_start_date ||= FormHandler.instance.lettings_forms["previous_lettings"].start_date
   end
 
-  def second_collection_end_date
-    @second_collection_end_date ||= FormHandler.instance.lettings_forms["current_lettings"].end_date
+  def previous_collection_end_date
+    @previous_collection_end_date ||= FormHandler.instance.lettings_forms["previous_lettings"].end_date
```

Comment:
these instantated start and end dates are the issues causing the tests to fail - their state is carried on between tests, e.g. in tasklist_helper_spec.rb: if "with 2023 deadline/returns relevant text" runs before "with 2024 deadline/returns relevant text", the 2024 one will fail because these start and end dates are still set from the previous test